web/src/scripts/containers/ProjectNavigator.jsx

```diff
@@ -117,6 +117,13 @@ class ProjectNavigator extends Component {
     textEditorCompositeActions.updateDecoEntryRequire(requireText)
   }
 
+  handleDrop = (sourceNode, targetNode) => {
+    this.props.fileTreeCompositeActions.renameFile(
```

Comment:
Interestingly, it seemed to work with dirs using just `renameFile`. But of course I'll update any way.